.github/ISSUE_TEMPLATE/module-bug-report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Module bug report
+about: Create a report to help improve a specific module
```

Comment:
Are these issue templates supposed to be in this PR?

Reply:
No. Depressing how they managed to sneak in, I thought I'd got rid. On second thoughts, maybe this PR was affected before I reverted that change from my master.